core/jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.ts

```diff
@@ -442,7 +442,7 @@ export class CreateServiceComponent implements OnInit {
           event["action"] = "PutRecord";
         } else if (this.eventExpression.type === "s3") {
           event["source"] = this.eventExpression.S3BucketName;
-          event["action"] = "S3:" + this.eventExpression.S3BucketName + ":*";
+          event["action"] = "s3:" + this.eventExpression.S3BucketName + ":*";
```

Comment:
this should not be bucket name. it should be just the actual action "s3:ObjectCreated:*"

Reply:
This is on an internal file and is currently irrelevant to oss Jazz

Reply:
but even in internal this will be a bug as this wont work. the action is not correct.

Reply:
in internal repo, this is handled correctly